float32 min_pwm
float32 max_pwm
float32 force_landing_thrust
uint8 pwm_conversion_mode
spinal/MotorInfo[] motor_info
